homeassistant/components/calendar/__init__.py

```diff
@@ -530,13 +538,17 @@ def state_attributes(self) -> dict[str, Any] | None:
         if (event := self.event) is None:
```

Comment:
I realize from yoru comment/screenshot: We shouldn't stop exporting the color because there are no upcoming events. It needs to be always present.

Reply:
done